lib/concurrent/immutable_struct.rb

```diff
@@ -71,20 +71,20 @@ def select(&block)
     end
 
     # @!macro struct_new
-    def self.new(*args, &block)
+    def self.new(*args, **kw_args, &block)
```

Comment:
It would be better to have the signature more specific `def self.new(*args, keyword_init: false, &block)`. Afaik no other keyword argument has any effect.